shared/build.gradle.kts

```
@@ -58,10 +61,33 @@ kotlin {
 }
 
 android {
-    compileSdk = 32
+    compileSdk = Android.CompileSdk
     sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
     defaultConfig {
-        minSdk = 26
-        targetSdk = 32
+        minSdk = Android.MinSdk
+        targetSdk = Android.CompileSdk
```

Ditto here on naming refactor:
```suggestion
        targetSdk = Android.TargetSdk
```